tests/models/siglip/test_tokenization_siglip.py

```diff
@@ -139,6 +136,10 @@ def siglip_tokenizer(self):
     def get_tokenizer(self, **kwargs) -> SiglipTokenizer:
         return self.tokenizer_class.from_pretrained(self.tmpdirname, **kwargs)
 
+    # Copied from tests.models.t5.test_tokenization_t5.T5TokenizationTest.get_rust_tokenizer with T5->Siglip
+    def get_rust_tokenizer(self, **kwargs) -> SiglipTokenizerFast:
+        return self.rust_tokenizer_class.from_pretrained(self.tmpdirname, **kwargs)
+
```

Comment:
One of the tests can be skipped as the tokenizer does lower case

Reply:
Which test can be skipped do you mean?

Reply:
`lower_case` related test IMO

Reply:
I skipped test_added_tokens_do_lower_case bc it handles lowercasing inherently